app/styles/_core.less

```diff
@@ -79,6 +79,13 @@
 
 /* Fix some styles with the angular-ui-select bootstrap theme that isn't right when combined with all of our styles */
 .ui-select-bootstrap {
+  /* Fixes ui-select's input box extending to 100% page width if the page has been resized after page-load
```

Comment:
Nice fix. Do you have a link to an issue with ui-select instead? It might be more helpful. If they ever fix it, we can eliminate this little hack.